### PR TITLE
Low pass filter perf profiler

### PR DESCRIPTION
**Pull request by carolhmj**

### CODE REVIEW

File: inspector/src/components/graph/canvasGraphService.ts [hunk 1]
```diff
@@ -356,25 +366,36 @@ export class CanvasGraphService {
                 const timestamp = this.datasets.data.at(this.datasets.startingIndices.at(pointIndex));
                 const value = this.datasets.data.at(valueIndex);
 
+                if (!prevValue) {
```
Comment (RaananW) [inline inspector/src/components/graph/canvasGraphService.ts:369]: can it be 0? is 0 a valid value?

**Review by RaananW — COMMENTED** [review]
LGTM! just a small question

File: inspector/src/components/graph/canvasGraphService.ts (hunk 1 above)
Comment (carolhmj) [reply inspector/src/components/graph/canvasGraphService.ts:369]: It is a valid value, thanks for catching that!

Comment (carolhmj) [reply inspector/src/components/graph/canvasGraphService.ts]: Updated!

**Review by RaananW — APPROVED** [review]

**Pull request merged**